packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -143,36 +104,6 @@ export const detachExperienceStyles = (experience: Experience): string | undefin
           continue;
         }
 
-        const defaultPatternDivStyles: Record<string, string> = Object.fromEntries(
-          Object.entries(buildCfStyles({}))
-            .filter(([, value]) => value !== undefined)
-            .map(([key, value]) => [toCSSAttribute(key), value]),
-        );
-
-        // I create a hash of the object above because that would ensure hash stability
-        const styleHash = md5(JSON.stringify(defaultPatternDivStyles));
-
-        // and prefix the className to make sure the value can be processed
-        const className = `cf-${styleHash}`;
-
-        for (const breakpointId of breakpointIds) {
-          if (!mediaQueriesTemplate[breakpointId].cssByClassName[className]) {
-            mediaQueriesTemplate[breakpointId].cssByClassName[className] =
-              toCSSString(defaultPatternDivStyles);
-          }
-        }
-
-        // When iterating over instances of the same pattern, we will iterate over the identical
-        // pattern nodes again for every instance. Make sure to not overwrite the values from previous instances.
-        if (!currentNode.variables.cfSsrClassName) {
-          currentNode.variables.cfSsrClassName = {
-            type: 'DesignValue',
-            valuesByBreakpoint: {
-              [breakpointIds[0]]: className,
-            },
-          };
-        }
-
```

Comment:
This chunk of code was the one adding the `cfSsrClassName` prop to the returned tree node.

Have I missed another place where the same thing was done? Without it, I think that style extraction will stop working and will always be done client side

Reply:
Yes, I have tested the PR and also talked to @anwaar931  who added this part of the code and who had more background knowledge. I tested, checked the code if it actually had an impact, and then asked him to confirm that this code had no effect on the function result. This logic created a class name independent of the current node and which all consists of default styles. We never actually use it which is why we can remove it.